tap-snapshots/test/lib/commands/doctor.js.test.cjs

```diff
@@ -9,124 +9,62 @@ exports[`test/lib/commands/doctor.js TAP all clear > logs 1`] = `
 Object {
   "error": Array [],
   "info": Array [
-    Array [
-      "Running checkup",
-    ],
-    Array [
-      "checkPing",
-      "Pinging registry",
-    ],
-    Array [
-      "getLatestNpmVersion",
-      "Getting npm package information",
-    ],
-    Array [
-      "getLatestNodejsVersion",
-      "Getting Node.js release information",
-    ],
-    Array [
-      "getGitPath",
-      "Finding git in your PATH",
-    ],
-    Array [
-      "getBinPath",
-      "Finding npm global bin in your PATH",
-    ],
-    Array [
-      "verifyCachedFiles",
-      "Verifying the npm cache",
-    ],
-    Array [
-      "verifyCachedFiles",
-      String(
-        Verification complete. Stats: {
-          "badContentCount": 0,
-          "reclaimedCount": 0,
-          "missingContent": 0,
-          "verifiedContent": 0
-        }
-      ),
-    ],
+    "doctor Running checkup",
```

Comment:
It looks like here we *gain* the `doctor` verb.  The `dist-tag` snapshots become even more of a curiosity.

Reply:
Same as the dist-tags comment. Likely a byproducts of how we're pulling logs in the tests.  Still, it'd be nice if the tests really reflected the full content.  We're taking a little too much on faith if we are relying on the side effect of our mock-log methods.

Reply:
The doctor tests are a special case because most of the logs were previously run through `npmlog.tracker`. I added the doctor verb to most logs in the command to be consistent.

Reply:
Also mock-logs is now much closer to what the user actually sees. Previously it was catching the actual arguments and tests would assert based on the raw args. Now it gets the full stringified output that was already been run through `util.format`.

Mock logs does do some filtering of the logs by level and prefix, but it most cases it still asserts on the raw strings that would get sent to `process.stderr.write`.